### PR TITLE
Added arweave in the publishing with hosting services page

### PR DESCRIPTION
**Pull request by AnaLoznianu**

### CODE REVIEW

**Comment by mariacarmina** [comment]
Should we hide some IDs from the screenshots?

**Comment by AnaLoznianu** [comment]
> Should we hide some IDs from the screenshots?

I don't think they are leaking anything so I wouldn't bother

**Review by mariacarmina — APPROVED** [review]
LGTM

**Comment by jamiehewitt15** [comment]
We should also update the initial part of this page:

1. The page description should mention Arweave given that it's one of the examples.
2. The first paragraph should mention that users can use either centralised storage or decentralised storage. At the moment it reads as if an HTTP URL is the only option.
3. I think it would work ni
4. The first screenshot should be updated as it's currently out of date. 

Also, some formatting changes:
1. `Overview` should be an H2 title
2. Rather than having `Hosting Services` as an H3 title, I would have two separate H2 titles: `Decentralised Hosting` and `Centralised Hosting`
3. `Arweave`, `AWS` and `Azure` should be H3 titles instead of H4 titles. 

The ideal behind these formatting changes is to get a the side menu to work better and nicely show the different sections of this page, which should help with navigation.

**Comment by AnaLoznianu** [comment]
Great suggestions @jamiehewitt15 👍 I've integrated them all and updated the PR. 
Please let me know if you see something off or if you have other things you think should be changed. 
Thanks 🙏

File: using-ocean-market/asset-hosting.md [hunk 1]
```diff
@@ -1,30 +1,67 @@
...
 ---
 
-# Publishing with Hosting Services
+## Overview
 
-### Overview
+To publish on the Ocean Marketplace, publishers must first host their assets. It is up to the asset publisher to decide where to host the asset. For example, a publisher can store the content on decentralized storage like Arweave or choose a centralized solution like their AWS server, private cloud server, or other third-party hosting services. Through publishing, the information required to access the asset is encrypted and stored as a part of DDO on the blockchain. Buyers don't have access directly to this information, but they interact with the Provider, which decrypts it and acts as a proxy to serve the asset. The DDO only stores the location of the file, which is accessed on-demand by the Provider. Implementing a security policy that allows only the Provider to access the file and blocks requests from other unauthorized actors is recommended. One of the possible ways to achieve this is to allow only the Provider's IP address to access the data. But, not all hosting services provide this feature. So, the publishers must consider the security features while choosing a hosting service.
```
Comment (jamiehewitt15) [inline using-ocean-market/asset-hosting.md:8]:
```suggestion
To publish on the Ocean Marketplace, publishers must first host their assets. It is up to the asset publisher to decide where to host the asset. For example, a publisher can store the content on decentralized storage like Arweave or choose a centralized solution like their AWS server, private cloud server, or other third-party hosting services. Through publishing, the information required to access the asset is encrypted and stored as a part of DDO on the blockchain. Buyers don't have access directly to this information, but they interact with the [Provider](https://github.com/oceanprotocol/provider#provider), which decrypts it and acts as a proxy to serve the asset. The DDO only stores the location of the file, which is accessed on-demand by the Provider. Implementing a security policy that allows only the Provider to access the file and blocks requests from other unauthorized actors is recommended. One of the possible ways to achieve this is to allow only the Provider's IP address to access the data. But, not all hosting services provide this feature. So, the publishers must consider the security features while choosing a hosting service.
```

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

**Pull request merged**